modules/src/ics23_commitment/commitment.rs

```diff
@@ -43,13 +43,19 @@ impl From<Vec<u8>> for CommitmentRoot {
 #[derive(Clone, Debug, PartialEq)]
 pub struct CommitmentPath;
 
-#[derive(Clone, Debug, PartialEq, Eq, Serialize)]
+#[derive(Clone, PartialEq, Eq, Serialize)]
 #[serde(transparent)]
 pub struct CommitmentProofBytes {
     #[serde(serialize_with = "crate::serializers::ser_hex_upper")]
     bytes: Vec<u8>,
 }
 
+impl fmt::Debug for CommitmentProofBytes {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "[...]")
```

Comment:
Can we use `crate::serializers::ser_hex_upper` to serialize the byte array to a string instead? Might be useful for debugging/

Reply:
ok, changed to be the same as for CommitmentRoot.